launchable/commands/helper.py

```diff
@@ -57,3 +59,19 @@ def find_or_create_session(
                 is_observation=is_observation,
             )
             return read_session(saved_build_name)
+
+
+def _check_observation_mode_status(session: str, is_observation: bool):
+    if is_observation is False:
+        return
+
+    client = LaunchableClient()
+    res = client.request("get", session)
+
+    # only check when the status code is 200 not to stop the command
+    if res.status_code == 200:
+        is_observation_in_recorded_session = res.json().get("isObservation", False)
+        if is_observation and is_observation_in_recorded_session is False:
+            click.echo(click.style(
+                "Warning: you set --observation option in this command but you need to set it `launchable record session` command", fg='yellow'),
```

Comment:
[nits]
I feel it is better to point out clearly that `--observation` option does not work if used in `launchable subset` command.